docs/oci-storage.md

```diff
@@ -0,0 +1,35 @@
+# OCI Storage
+
+## Summary
+
+Fleet stores by default the bundle resources in etcd twice. This is done via the k8s API and there is a size limit, depending on the etcd configuration.
+
+This feature will allow users to choose an OCI registry as storage for bundle resources. The bundle resource will have an empty resource list and a reference to the OCI repository server. The bundledeployment will not point to a content resource, but to an OCI repository server instead.
+
+When using this feature the bundle resources are stored once, in the configured OCI registry, and `Fleet` won't be tied to possible `etcd` size limitations.
+
+This might me interesting for users that either need to store big `Bundles` and could also be seen as the first step for a `OCIOps` feature in the future.
```

Comment:
```suggestion
This may be interesting for users who need to store big `Bundles`, and could also be seen as the first step for an `OCIOps` feature in the future.
```